packages/web3/test/integration/web3.accounts.test.ts

```diff
@@ -77,8 +77,12 @@ describe('web3.accounts', () => {
 					}),
 				).resolves.toBeDefined();
 
+				const txWithGas = {
+					...tx,
+					gasPrice: '0x271000',
```

Comment:
added `gasPrice` explicitly in tests of this file, because now `signTransaction` will not fill gas price and validation fails if no gasprice or gas0x2 params are provided .